src/utils.ts

```diff
@@ -179,5 +179,10 @@ export const parseControllers = (controllersPath: string, params: ParseControlle
   // Write the import statements
   const contents = definitions.map((definition) => `import ${definition.name} from '${definition.path}';`).join('');
 
+  // Replace all _ with - in identifiers
+  for (const definition of definitions) {
+    definition.identifier = definition.identifier.replace(/_/g, '-');
```

Comment:
This is technically a breaking change if anyone was using `directorySeparator` before with `_`, even if it's highly unlikely. I think only the name of the controller should be replaced in this PR, and I'll make another PR to completely remove this option.

Reply:
Although, thinking about it now, even just changing the name **is** technically a breaking change.

I'm not sure if this is a bug, I'd say it's rather a misconception, and a new (major?) release should be done to fix it. What do you think ?

Reply:
I'm not entirely sure about the details of this change, so feel free to close the pull request and make another one if needed, since it's your repo. It's important that it aligns with your vision and understanding. I've just made some workarounds to make it work in my project. Let me know if there's anything else you need from me!